<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form_ingresos_diarios</class>
 <widget class="QWidget" name="form_ingresos_diarios">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>888</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ingresos Diarios</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 75 10pt &quot;KacstOne&quot;;
selection-background-color: rgb(255, 170, 127);
color: rgb(0, 0, 0);
background-color: rgb(81, 1, 1);
selection-color: rgb(0, 0, 0);
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(239, 235, 231);
</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_2">
      <attribute name="icon">
       <iconset>
        <normaloff>Íconos/documento-financiero.png</normaloff>Íconos/documento-financiero.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Ingresos Diarios</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="styleSheet">
          <string notr="true">QGroupBox{
background-color: rgb(239, 235, 231);
}</string>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>Fecha de movimientos a buscar :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateEdit" name="dte_fecha_ingresos_mov">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>KacstOne</family>
                <pointsize>10</pointsize>
                <weight>9</weight>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 255, 255);</string>
              </property>
              <property name="displayFormat">
               <string>dd/M/yyyy</string>
              </property>
              <property name="calendarPopup">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="boton_ingresos_buscar">
              <property name="font">
               <font>
                <family>KacstOne</family>
                <pointsize>10</pointsize>
                <weight>9</weight>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(251, 204, 193);
</string>
              </property>
              <property name="text">
               <string>Buscar</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Registro de Ingresos diarios</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="styleSheet">
            <string notr="true">QGroupBox{
background-color: rgb(239, 235, 231);

}
</string>
           </property>
           <property name="title">
            <string/>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="0" column="0">
               <widget class="QTableWidget" name="tw_ingresos_registros">
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(251, 204, 193);</string>
                </property>
                <property name="selectionMode">
                 <enum>QAbstractItemView::MultiSelection</enum>
                </property>
                <property name="selectionBehavior">
                 <enum>QAbstractItemView::SelectRows</enum>
                </property>
                <attribute name="horizontalHeaderDefaultSectionSize">
                 <number>150</number>
                </attribute>
                <column>
                 <property name="text">
                  <string>Nombre y Apellido</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Cuota</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Número </string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Importe</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Punitorios</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Descuento</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Estado</string>
                 </property>
                </column>
               </widget>
              </item>
              <item row="1" column="0" alignment="Qt::AlignHCenter">
               <widget class="QPushButton" name="boton_ingresos_imprimir">
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(251, 204, 193);
</string>
                </property>
                <property name="text">
                 <string>Imprimir</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>N° Planilla :</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lne_ingresos_nro_planilla">
                  <property name="styleSheet">
                   <string notr="true">background-color: rgb(255, 255, 255);</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>Total de Ingresos del Día: </string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lne_ingresos_total">
                  <property name="styleSheet">
                   <string notr="true">background-color: rgb(255, 255, 255);</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>dte_fecha_ingresos_mov</tabstop>
  <tabstop>boton_ingresos_buscar</tabstop>
  <tabstop>tw_ingresos_registros</tabstop>
  <tabstop>boton_ingresos_imprimir</tabstop>
  <tabstop>lne_ingresos_nro_planilla</tabstop>
  <tabstop>lne_ingresos_total</tabstop>
  <tabstop>tabWidget_2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
